idx/idx.go

```diff
@@ -147,6 +147,17 @@ type MetricIndex interface {
 	// LastUpdate time is >= the given value.
 	Tags(int, string, int64) ([]string, error)
 
+	// AutoCompleteTags is used to generate a list of possible values that could
```

Comment:
`s/is used to generate/generates/`

Reply:
https://github.com/grafana/metrictank/pull/779/commits/9d217cd83c383b8046976e88be2a0f6c43946d2f